tests/lib/tools/tests.session

```diff
@@ -148,7 +156,7 @@ case "$action" in
 			# Because some systems do not support systemd --user (see
 			# tests/main/tests.session-support for details), check if this is
 			# expected to work ahead of trying.
-                        if tests.session -u "$u" exec systemctl --user is-enabled default.target >/dev/null; then
+      if tests.session -u "$u" exec systemctl --user is-enabled default.target >/dev/null; then
```

Comment:
Tabs/spaces?

Reply:
@zyga I already fixed some indentations in that file